src/condor_tests/test_htcondor_cli_sys_status.py

```diff
@@ -0,0 +1,183 @@
+    # Verify Schedd 1 rdcdc -> 28 HP
+    {
+        "MyAddress" : "FILLER",
+        "Machine" : "LOCAL_HOST",
+        "MyType" : "Scheduler",
+        "RecentDaemonCoreDutyCycle" : 1,
+        "ExpectedDaemon" : "SCHEDD",
+        "ExpectedHP" : 28,
+    },
+    # Verify Schedd with 0 bytes per second Upload -> 93
+    {
+        "MyAddress" : "FILLER",
+        "Machine" : "LOCAL_HOST",
+        "MyType" : "Scheduler",
+        "RecentDaemonCoreDutyCycle" : 0,
+        "TransferQueueNumUploading" : 1,
+        "FileTransferUploadBytesPerSecond_1m" : 0,
+        "ExpectedDaemon" : "SCHEDD",
+        "ExpectedHP" : 93,
+    },
+    # Verify Download MB Waiting < 100 -> 100 HP
+    {
+        "MyAddress" : "FILLER",
+        "Machine" : "LOCAL_HOST",
+        "MyType" : "Scheduler",
+        "RecentDaemonCoreDutyCycle" : 0,
+        "FileTransferMBWaitingToDownload" : 10,
+        "TransferQueueDownloadWaitTime" : 100000,
+        "TransferQueueNumWaitingToDownload" : 3,
+        "ExpectedDaemon" : "SCHEDD",
+        "ExpectedHP" : 100,
+    },
+    # Verify Download MB Waiting > 100 && Wait time < 5m -> 100 HP
+    {
+        "MyAddress" : "FILLER",
+        "Machine" : "LOCAL_HOST",
+        "MyType" : "Scheduler",
+        "RecentDaemonCoreDutyCycle" : 0,
+        "FileTransferMBWaitingToDownload" : 250,
+        "TransferQueueDownloadWaitTime" : 20,
+        "TransferQueueNumWaitingToDownload" : 3,
+        "ExpectedDaemon" : "SCHEDD",
+        "ExpectedHP" : 100,
+    },
+    # Verify Download MB Waiting > 100 && Wait time > 5m -> 93 HP
+    {
+        "MyAddress" : "FILLER",
+        "Machine" : "LOCAL_HOST",
+        "MyType" : "Scheduler",
+        "RecentDaemonCoreDutyCycle" : 0,
+        "FileTransferMBWaitingToDownload" : 250,
+        "TransferQueueDownloadWaitTime" : 100000,
+        "TransferQueueNumWaitingToDownload" : 3,
+        "ExpectedDaemon" : "SCHEDD",
+        "ExpectedHP" : 93,
+    },
+]
+
+@action(params={name: name for name in TEST_CASES})
+def run_commands(default_condor, request):
+    with default_condor.use_config():
+        # HTCondor cli logger outputs to stderr?
+        p = subprocess.run(TEST_CASES[request.param], stdout=subprocess.PIPE, stderr=subprocess.PIPE, timeout=60)
+        output = p.stdout.rstrip().decode() + p.stderr.rstrip().decode()
+        yield output
+
+class TestHTCondorCLISytemStatus:
+    def test_convert_ad_unit_tests(self):
+        for test_ad in UNIT_TEST_CASES:
+            display = "\nFailed Unit Test Input Ad: {{\n{0}\n}}\n".format('\n'.join([f'\t{key} : {val}' for key, val in test_ad.items()]))
+            result = _ad_to_daemon_status(test_ad)
+            if "MyType" not in test_ad or test_ad["MyType"] == "Shadow":
+                assert result is None, display
+            else:
+                daemon, status = result
+                assert test_ad["ExpectedDaemon"] == daemon, display
+                assert test_ad["ExpectedHP"] == int(status["HealthPoints"]), display
```

Comment:
It's not necessary, but it may well be easier to read in case of failures if you let PyTest execute the loop, instead.  It's also conventional to avoid globals unless you need them for something:
```
#
# Approximately...
#

@action(params=UNIT_TEST_CASES)
def test_tuple(requests):
     return (request.param, UNIT_TEST_CASES[request.param])

@action
def test_case(test_tuple):
    return test_tuple[1]

@action
def test_name(test_tuple):
    return test_tuple[0]

@action
def unit_test_result(test_case):
    return _ad_to_daemon_status(test_case)

# more approximation
def test_convert_ad_unit_tests(self, test_name, test_case, unit_test_result):
    assert test_case["ExpectedDaemon"] == result[0], display
    assert test_case["ExpectedHP"] == int(result[1]["HealthPoints"], display